### PR TITLE
fix: low browser support

### PR DESCRIPTION
**Pull request by poyoho**
### 🔗 Linked issue

compatible with lower version browsers. `?.`

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation or readme)
- [x] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Review by danielroe — COMMENTED** [review]
This looks like a good change. 👍 

However, we are currently discussing whether to move Nuxt Bridge to support modern formats only, which will possibly mean a hard break with IE11: https://github.com/nuxt/bridge/issues/15.

I think I'd be happy to merge this until (if) we get to that point, but wdyt @pi0?

**Comment by pi0** [comment]
Current/Nuxt2 baseline for **modern** is supporting modules (Chrome and FF ~60) and optional chaining is more recent (chrome 80 and FF 74). 

I think it is worth checking how many other libraries need to transform to make this happen. If this only change makes compatibility, I think it worth.

File: src/vite/templates.ts [hunk 1]
```diff
@@ -138,7 +138,7 @@ function mergeProperty (storeModule, moduleData, property) {
 export const clientConfigTemplate = {
   filename: 'nitro.client.mjs',
   getContents: () => `
-export const useRuntimeConfig = () => window?.__NUXT__?.config || {}
+export const useRuntimeConfig = function(){return window && window.__NUXT__ && window.__NUXT__.config || {}}
```
Comment (danielroe) [inline src/vite/templates.ts:141]: Unlike the other two files, this code will be processed by rollup/vite. Shouldn't features like optional chaining or arrow functions be compiled out if the right target is specified?

Comment (pi0) [reply src/vite/templates.ts:141]: Additionally, browser support for arrow functions is below modern baseline (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions).

**Comment by poyoho** [comment]
the app plugin must be using. It better to move it to `Object.defineProperty`?

And the `useRuntimeConfig` is seems can to move to `Object.defineProperty`.

**Comment by pi0** [comment]
@poyoho Please make sure to check MDN for browser support. The lowest browser we can aim for bridge is **chrome and firefox ~60** that support module scripts. They both support [Proxy](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy) on an earlier version. More precise chart: https://caniuse.com/es6-module-dynamic-import

**Comment by poyoho** [comment]
But ie no support it https://caniuse.com/?search=proxy 👀

**Comment by pi0** [comment]
Yes. We won't support internet explorer with a bridge. The least base line for bridge is browsers listed in https://caniuse.com/es6-module-dynamic-import.

**Review by danielroe — APPROVED** [review]

**Comment by poyoho** [comment]
@pi0 It better to declare in the document 😅. The lowest browser we can using bridge is chrome and firefox ~60

File: README.md [hunk 2]
```diff
@@ -6,7 +6,9 @@ Bridge is a forward-compatibility layer that allows you to experience many of th
 
 Using Nuxt Bridge, you can make sure your project is (almost) ready for Nuxt 3 and have the best developer experience without needing a major rewrite or risk breaking changes.
 
-⚠️ **Note:** Nuxt Bridge provides identical features to Nuxt 3 ([Nuxt 3 docs](https://v3.nuxtjs.org/guide/features/views)) but there are some limitations, notably that `useAsyncData` and `useFetch` composables are not available. Please read the rest of this page for details.
+⚠️ **Note:** Nuxt Bridge provides identical features to Nuxt 3 ([Nuxt 3 docs](https://v3.nuxtjs.org/guide/features/views)) but there are some limitations, notably that `useAsyncData` and `useFetch` composables are not available. Please read the rest of this page for details. 
+
+⚠️ **Note:** Nuxt Bridge don't plan support internet explorer. The least base line for bridge is browsers listed in https://caniuse.com/es6-module-dynamic-import.
```
Comment (danielroe) [inline README.md:11]:
```suggestion
⚠️ **Note:** Nuxt Bridge does not support Internet Explorer. Supported browsers are listed at https://caniuse.com/es6-module-dynamic-import.
```

**Comment by poyoho** [comment]
It plan to release?

**Comment by danielroe** [comment]
Thank you! :heart: This should now be available on edge releases.

**Pull request merged**